### ISSUE TITLE
c-lightning support

### ISSUE THREAD

**Issue by MrManPew** [stakwork/sphinx-relay#155]
Hi,
Any planes to support [c-lightning](https://github.com/ElementsProject/lightning)?

**Comment by pitoi** [comment]
Yes, we've had a few planning calls in the last month but no timeline yet.

**Comment by LightningHelper** [comment]
any update? 
Maybe this can help?
https://github.com/Ride-The-Lightning/c-lightning-REST

**Comment by pitoi** [comment]
We just finished a version that supports the Greenlight specific GRPC to c-lightning. We'll tackle direct linking to c-lightning after we get Greenlight working smoothly.

**Comment by LightningHelper** [comment]
Great!
